winui/Shadow/Getting-Started.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: Getting Started with WinUI Shadow control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Shadow(SfShadow) control, its elements, and more.
+platform: winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# Getting Started with WinUI Shadow
+
+This section explains the steps required to add the [WinUI Shadow](https://www.syncfusion.com/winui-controls/shadow) control. This section covers only basic features needed to get started with Syncfusion `Shadow` control.
+
+## Creating an application with WinUI Shadow
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to [Syncfusion.Core.WinUI](https://www.nuget.org/packages/Syncfusion.Core.WinUI) NuGet. 
+3. Import the control namespace `Syncfusion.UI.Xaml.Core` in XAML or C# code.
+4. Initialize the `SfShadow` control.
+
+
+{% tabs %}
+{% highlight xaml %}
+
+<Page
+    x:Class="GettingStarted.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:GettingStarted"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:syncfusion="using:Syncfusion.UI.Xaml.Core"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid>
+       <syncfusion:SfShadow Name="buttonShadow"
+                      Color="#40000000"
+                      BlurRadius="8"
+                      OffsetX="4"
+                      OffsetY="4"
+                      ShadowCornerRadius="5">
+          <Button x:Name="buttonShadow" Height="50" width="100" Content="Button"         
+              CornerRadius="5"/>
+       </syncfusion:SfShadow>
+    </Grid>
+</Page>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.UI.Xaml.Core;
+
+namespace GettingStarted
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+           this.InitializeComponent();
+           // Creating an instance of the Shadow control.
+           SfShadow shadow = new SfShadow();
+
+           //Setting Color, BlurRadius, OffsetX, OffsetY, ShadowCornerRadius of Shadow.
```

Comment:
@Priyanka-Vijayasankar Add Content code in C#.

Reply:
Added content code in C#.